web/src/lib/components/forms/tag-asset-form.svelte

```diff
@@ -38,6 +39,13 @@
 </script>
 
 <FullScreenModal title={$t('tag_assets')} icon={mdiTag} onClose={onCancel}>
+  <div class="text-sm">
+    <p>
+      {$t('tag_not_found_question')}
+      {$t('create')}
+      <a href={AppRoute.TAGS} class="text-immich-primary dark:text-immich-primary underline">{$t('here')}</a>
```

Comment:
This should be one translation. By splitting strings like this, the context of the sentence is lost and it prevents changing the word order.

Reply:
Done